app/src/main/java/com/example/bikini_android/ui/adding/feed/AddingFeedFragment.kt

```diff
@@ -0,0 +1,40 @@
+/*
+ * AddingFeedFragment.kt 2020. 11. 1
+ *
+ * Copyright 2020 BasicBug. All rights Reserved.
+ *
+ */
+
+package com.example.bikini_android.ui.adding.feed
+
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.databinding.DataBindingUtil
+import com.example.bikini_android.R
+import com.example.bikini_android.databinding.FragmentAddingFeedBinding
+import com.example.bikini_android.ui.holder.MainHolderFragment
+
+/**
+ * @author MyeongKi
+ */
+
+class AddingFeedFragment :MainHolderFragment(){
+    private lateinit var binding: FragmentAddingFeedBinding
+
+    override fun onCreateView(
+        inflater: LayoutInflater,
+        container: ViewGroup?,
+        savedInstanceState: Bundle?
+    ): View? {
```

Comment:
나 회사에서는 아래처럼 하기도 하는데
```suggestion
    ): View? = DataBindingUtil.inflate<FragmentAddingFeedBinding>(inflater, R.layout.fragment_adding_feed, container, false).also {
        binding = it
    }.root
```
여기서 DatabindingUtil ktx 를 쓰면 저렇게 DatabindingUtil 이라고 쓰지 않아도 되서 더 짧게 사용할 수 있었던 것 같아